R/outliers.R

```diff
@@ -35,10 +37,11 @@ is_outlier <- function(x, method = "iqr", ...) {
     all(is.finite(x) | is.na(x)) # All values are finite OR are NA
   })
 
-  out <- switch(method,
-    "quantile" = quantile_outlier(x, na.rm = TRUE, ...),
-    "iqr" = iqr_outlier(x, na.rm = TRUE, ...)
-  )
+  if (method == "quantile") {
+    out <- quantile_outlier(x, probs = probs)
+  } else {
+    out <- iqr_outlier(x, mult = mult)
+  }
```

Comment:
No `ifelse` here since we're returning a vector, in which case `ifelse` coerces the return value to the same shape as the test.